src/components/Character.js

```diff
@@ -1,7 +1,22 @@
 import React from "react";
+import styled from "styled-components";
+import PropTypes from 'prop-types';
+
+const StyledChar = styled.li `
```

Comment:
You might want to add styling with styled-components after your component return